src/app/routes/utils/fetchPageData/utils/getRouteProps/index.test.js

```diff
@@ -130,6 +130,12 @@ describe('getRouteProps', () => {
         match,
         route,
         service: 'fallbackService',
+        assetUri: undefined,
```

Comment:
Still trying to figure out previously why this value wasn't returned at all, whereas now it is with 'undefined'.

Same goes for 

`errorCode`, `id` & `variant` (non of those values were changed in Jonathan's OG PR)